tests/tests/cpld_bmarkfrac_v16_nsst

```diff
@@ -0,0 +1,150 @@
+                   RESTART/phy_data.tile6.nc \
+                   RESTART/sfc_data.tile1.nc \
+                   RESTART/sfc_data.tile2.nc \
+                   RESTART/sfc_data.tile3.nc \
+                   RESTART/sfc_data.tile4.nc \
+                   RESTART/sfc_data.tile5.nc \
+                   RESTART/sfc_data.tile6.nc \
+                   RESTART/MOM.res.nc \
+                   RESTART/MOM.res_1.nc \
+                   RESTART/MOM.res_2.nc \
+                   RESTART/MOM.res_3.nc \
+                   RESTART/iced.2013-04-01-43200.nc \
+                   RESTART/ufs.cpld.cpl.r.2013-04-01-43200.nc"
+
+export_fv3
+export_cpl
+
+export SYEAR="2013"
+export SMONTH="04"
+export SDAY="01"
+export SHOUR="00"
+
+export DAYS="0.5"
+export FHMAX="12"
+export RESTART_INTERVAL="6"
+export RESTART_N=${RESTART_INTERVAL}
+
+export TASKS=$TASKS_cpl_bmrk
+export TPN=$TPN_cpl_bmrk
+export INPES=$INPES_cpl_bmrk
+export JNPES=$JNPES_cpl_bmrk
+export THRD=$THRD_cpl_bmrk
+export WRTTASK_PER_GROUP=$WPG_cpl_bmrk
+
+export med_petlist_bounds=$MPB_cpl_bmrk
+export atm_petlist_bounds=$APB_cpl_bmrk
+export ocn_petlist_bounds=$OPB_cpl_bmrk
+export ice_petlist_bounds=$IPB_cpl_bmrk
+
+# atm/ocn/ice resolution
+export ATMRES='C384'
+export NPX=385
+export NPY=385
+export IMO=1536
+export JMO=768
+
+export OCNRES='025'
+export ICERES='0.25'
+export NX_GLB=1440
+export NY_GLB=1080
+export NPROC_ICE='48'
+
+# resolution dependent setting
+export CDMBWD=${CDMBWD_c384}
+
+# set component and coupling timesteps
+export DT_ATMOS='225'
+export DT_CICE=${DT_ATMOS}
+export DT_DYNAM_MOM6='900'
+export DT_THERM_MOM6='1800'
+export CPL_SLOW=${DT_THERM_MOM6}
+export CPL_FAST=${DT_ATMOS}
+
+# nems.configure
+export coupling_interval_slow_sec=${CPL_SLOW}
+export coupling_interval_fast_sec=${CPL_FAST}
+
+export FRAC_GRID='.T.'
+export CPLMODE='nems_frac'
+
+export NPZ=127
+export NPZP=128
+export PSM_BC=1
+export DDDMP=0.2
+
+# resolution dependent files
+export MOM_INPUT="MOM_input_template_${OCNRES}"
+export MESHOCN_ICE="mesh.mx${OCNRES}.nc"
+export CICEGRID="grid_cice_NEMS_mx${OCNRES}.nc"
+export CICEMASK="kmtu_cice_NEMS_mx${OCNRES}.nc"
+export CHLCLIM="seawifs-clim-1997-2010.${NX_GLB}x${NY_GLB}.v20180328.nc"
+export FRUNOFF="runoff.daitren.clim.${NX_GLB}x${NY_GLB}.v20180328.nc"
+
+export FNALBC="'global_snowfree_albedo.bosu.t766.1536.768.rg.grb',"
+export FNVETC="'global_vegtype.igbp.t766.1536.768.rg.grb',"
+export FNSOTC="'global_soiltype.statsgo.t766.1536.768.rg.grb',"
+export FNSMCC="'global_soilmgldas.statsgo.t766.1536.768.grb',"
+export FNABSC="'global_mxsnoalb.uariz.t766.1536.768.rg.grb',"
+
+export OZ_PHYS_NEW=".T."
+
+export MOM6_RIVER_RUNOFF='True'
+export MOM6_RESTART_SETTING="r"
+export MOM6_REPRO_LA='True'
+
+export RUNID="cpcice"
+
+export INPUT_NML=input.benchmark_v16.nml.IN
+
+export FIELD_TABLE="field_table_gfsv16"
+export DIAG_TABLE="diag_table_bmark_v16_template"
+export SUITE_NAME="FV3_GFS_v16_couplednsst"
+export nstf_name=2,1,0,0,0
```

Comment:
You need to make nstf_name in the file input.benchmark_v16.nml.IN to be a variable, which is defined here in the test. So, here define: NSTF_NAME='2,1,0,0,0'. Then in input.benchmark_v16.nml.IN set nstf_name = @[NSTF_NAME]. This will also need to be done in the same way for the control_nsst test that you added.

Reply:
> You need to make nstf_name in the file input.benchmark_v16.nml.IN to be a variable, which is defined here in the test. So, here define: NSTF_NAME='2,1,0,0,0'. Then in input.benchmark_v16.nml.IN set nstf_name = @[NSTF_NAME]. This will also need to be done in the same way for the control_nsst test that you added.

@DeniseWorthen: Yes, it is better, since it is consistent with all the places of nstf_name in the parm/input*nml files. I change NSTF_NAME to be nstf_name since "  nstf_name    = 0,0,0,0,0" has been defined in input.benchmark_v16.nml.IN, I will remove this, and add nstf_name = @[NSTF_NAME] in input.benchmark_v16.nml.IN, then I can keep the upper case NSTF_NAME='2,1,0,0,0' in cpld_bmarkfrac_v16_nsst, as I did for cpld_control_nsst. This is the first time for me to create a test, a good experience.